packages/components/src/index.css

```diff
@@ -0,0 +1,3 @@
+@import url("@hopper-ui/icons/index.css");
```

Comment:
I'm pretty sure there was a reason to split these, otherwise we would have gone this route.